githubteacher-octovillage.md

```diff
@@ -0,0 +1,2 @@
+# GitHub Teacher - Octovillage
+This is the hometown of the @githubteacher. It contains some much needed documentation to explain the @githubteacher.
```

Comment:
Would be great to include your picture.

Otherwise it looks ✨ . :shipit: 

:octocat: :sparkling_heart: